doc/specs/#1564 - Settings UI/cascading-settings.md

```diff
@@ -0,0 +1,166 @@
+
+#### 2: \<inherit\> option
+
+Each setting is an Editable ComboBox (except for boolean and enumerable settings). For booleans, the items will be Enabled and Disabled in a regular ComboBox. Enumerable settings will have their options listed in a regular ComboBox. For integers, most commonly used numbers will be listed.
+
+![Dropdown inheritance](./inheritance-dropdown.png)
+
+**Pros**
+
+- Doesn't clutter the screen.
+
+**Cons**
+
+- Every setting is a dropdown.
+
+**Pitfalls**
+
+- How will color pickers work with this scenario?
+
+**Other considerations**
+
+Each dropdown has either "inherit" or "custom". If the user selects "custom", the original control will appear (i.e. a color picker for colors or a number picker for integers).
+
+This option was not chosen because it added too much overhead for changing a setting. For example, if you wanted to enable acrylic, you'd have to click the dropdown, select custom, watch the checkbox appear, and then select the checkbox.
+
+#### 3: Lock Button
+
+Every setting will have a lock button next to it. If the lock is locked, that means the setting is being inherited from Global, and the control is disabled. If the user wants to edit the setting, they can click the lock, which will changed it to the unlocked lock icon, and the control will become enabled.
+
+![Locks inheritance](./inheritance-locks.png)
+
+**Pros**
+
+- Least amount of clutter on the screen while still keeping the original controls
+
+**Cons**
+
+- The lock concept is slightly confusing. Some may assume locking the setting means that it *won't* be inherited from Global and that it's "locked" to the profile. This is the opposite case for its current design. However, flipping the logic of the locks wouldn't make sense with an unlocked lock and a disabled control.
+
+### Partial Parity with JSON
+
+The settings UI does not necessarily need to have full parity with JSON. Below are proposals for partial parity with JSON, while also providing the simplicity of the settings UI.
+
+#### Add New --> Duplicate Profile
+
+The Add new profile button in the navigation menu would take you to a new page. This page will have radio buttons listing your profiles along with a default settings option. The user can choose to either duplicate a profile or create a new one from the default settings. Once the user makes a selection, the settings UI will take them to their new profile page. The fields on that profile page will be filled according to which profile selection the user made.
+
+![Add new profile](./add-new-profile.png)
+
+#### Reset Profile button
+
+On the Advanced pivot of a profile's page, there will be a button at the bottom for resetting a profile called "Reset to default settings". This button will remove the user's custom settings inside this profile's object and reset it to defaults.

```

Comment:
> reset it to defaults

What does this mean? defaults.json? profile.defaults?

What happens when you click this button on a Profile that doesn't come from defaults.json or the dynamic profile generators?